diff --git a/src/XrdSsi/XrdSsiFileReq.cc b/src/XrdSsi/XrdSsiFileReq.cc
index 4f6121ff7..f305c4c5d 100644
--- a/src/XrdSsi/XrdSsiFileReq.cc
+++ b/src/XrdSsi/XrdSsiFileReq.cc
@@ -469,11 +469,11 @@ void XrdSsiFileReq::Finalize()
    switch(urState)
          // Request is being scheduled, so we can simply abort it.
          //
-         {case isNew:    DEBUGXQ("Aborting request processing");
-                         urState = isAbort;
+         {case isNew:    urState = isAbort;
                          cbInfo  = 0;
                          sessN   = "???";
                          Stats.Bump(Stats.ReqAborts);
+                         DEBUGXQ("Aborting request processing");
                          return;
                          break;
 
